cmd/meroxa/root/apps/run.go

```diff
@@ -105,6 +111,9 @@ func (r *Run) Execute(ctx context.Context) error {
 		}
 		return r.turbineCLI.Run(ctx)
 	case "rb", turbine.Ruby:
+		if err = builder.CheckFeatureFlag(r, r); err != nil {
```

Comment:
Oh oops, forgot to mention, this breaks the run test for ruby since feature flag blocks it